lib/adequate_crypto_address/eth.rb

```diff
@@ -1,83 +1,83 @@
 # frozen_string_literal: true
 
 module AdequateCryptoAddress
-  class Eth
-    attr_reader :address, :raw_address
+    class Eth
```

Comment:
This class is very.. complex for an address validator

I get the normalize method, but do we really need to do checksum and keccak256 stuff?